pages/api/graphql/schema/types/TestimonialType.ts

```diff
@@ -0,0 +1,17 @@
+import { gql } from 'apollo-server-micro'
+
+module.exports = gql`
+
+  type Testimonial {
+    id: Int
+    person_name: String
+    testimony: String
+    person_image_url: String
+    project: [Project]
```

Comment:
There is an incorrect relationship here. A Testimonial should only ever have 1 project (can only have a single `project_id` so why are we returning an array of Projects? especially since the field name is `project` singular

This needs to be fixed so that the frontend does not have to work with an array of projects coming in from the API